backend/README.md

```diff
@@ -12,3 +12,9 @@ The API is configured to run in a Docker container and has the following depende
 - [DAL](./docs/DAL.md) - Data Access Layer information.
 - [Tools](./docs/TOOLS.md) - Tooling information.
 - [Versioning](./docs/VERSIONING.md) - Version information.
+
+## Health Checks UI
+
+Health checks are configured to run against /api/live and /api/ready endpoints. These correspond to the kubernetes concepts of liveliness and readiness, see [here](https://docs.openshift.com/container-platform/3.11/dev_guide/application_health.html). 
```

Comment:
It isn't critical lol, but I create a `HealthController`, perhaps the endpoints should be `/api/health/live` and `/api/health/ready` instead?